### ISSUE TITLE
Move relevant things from GitOps WG repo to OpenGitOps project repo

### ISSUE THREAD

**Issue by scottrigby** [open-gitops/project#2]
## Update Jan 16 2022
The https://github.com/gitops-working-group/gitops-working-group repo has been superseded by https://github.com/cncf/tag-app-delivery/tree/main/gitops-wg/ and https://github.com/open-gitops/project/.
- PRs can not be moved between repos, so if still relevant they should be re-opened in these new repo locations
- All issues will be moved to the appropriate repo, or closed with a note on next steps the OP should take
- Discussions relevant to OpenGitOps will be moved to https://github.com/open-gitops/project/

## To-do
- [x] Licences (see #5) This PR needs to be merged first
- [x] entrypoint README (see #6)
- [x] #20
- [x] MAINTAINERS file #3
- [x] any other top-level information about the project
- [x] Close all PRs with note
- [x] Update OpenGitOps media team to match https://github.com/gitops-working-group/gitops-working-group/issues/108
    - Everyone was invited. Some invites still need to be accepted
- [x] Move any relevant issues or close with note

## Related
- Part of #1
- per https://github.com/gitops-working-group/gitops-working-group/discussions/147
- Prior planning, see https://github.com/orgs/gitops-working-group/projects/6

# Follow-up
- #9
- #12
- Do we want to add linting GitHub Workflow to OpenGitOps repos? See https://github.com/gitops-working-group/gitops-working-group/issues/45 and https://github.com/gitops-working-group/gitops-working-group/pull/59

**Comment by scottrigby** [comment]
Hey everyone. Finally finishing moving the old WG repo things issues over to OpenGitOps and App Delivery TAG repo. If you happen to look during this process, will be transferring the old WG repo to the open-gitops gh project temporarily, in order to properly move issues, discussions, etc. Keeping the issue description checklist updated for transparency ⏳

**Comment by scottrigby** [comment]
This is complete! ✅
